lib/google_maps_geocoder/google_maps_geocoder.rb

```diff
@@ -3,6 +3,8 @@
 require 'net/http'
 require 'rack'
 
+require 'google_maps_geocoder/google_maps_geocoder_error_handler'
```

Comment:
not sure what do you think about adding new classes here, since before it was just one class, but the length of this class has started to be a problem for rubocop